src/Entity/Server.php

```diff
@@ -162,7 +172,11 @@ public function configuration() {
     $manager = \Drupal::service('plugin.manager.graphql.schema');
 
     /** @var \Drupal\graphql\Plugin\SchemaPluginInterface $plugin */
-    $plugin = $manager->createInstance($this->get('schema'));
+    $plugin_configuration = [];
+    if (!empty($this->get('schema_configuration')) && !empty($this->get('schema_configuration')[$this->get('schema')])) {
```

Comment:
Instead of doing all these if checks, just instantiate the plugin without config, then check if it's an instanceof ConfigurableInterface and pass the configuration to `$plugin->setConfiguration()`. Also, can you assign `$this->get('schema_configuration')` and `$this->get('schema')` to variables if you need to re-use them more than once? :-)